module-domain/src/main/kotlin/site/yourevents/guest/port/in/OwnerNicknameUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package site.yourevents.guest.port.`in`
+
+import site.yourevents.guest.domain.Guest
+import java.util.UUID
+
+interface OwnerNicknameUseCase {
+    fun createOwnerNickname(
```

Comment:
해당 usecase는 주최자의 닉네임 뿐만 아니라, 참석 여부를 확인하는 로직에서도 재사용이 가능해보입니다!
```suggestion
interface GuestUseCase {
    fun createGuest(
```
따라서 클래스명과 메서드명을 다음과 같이 바꾸는 것은 어떻게 생각하시나요?

Reply:
좋은 방법인 것 같습니다!! 저는 제가 구현하던 부분만 생각하느라 이름을 OwnerNicknameUseCase로 지었네요 ㅎㅎ 수정하였습니다.